src/views/css/packlink-core-override.css

```diff
@@ -0,0 +1,17 @@
+#pl-page .pl-form-group input,
+#pl-page .pl-form-group select {
+  height: 100%;
+}
+
+#pl-pricing-policy-form .pl-form-group {
+  height: 100%
```

Comment:
It seems that this is not needed.

Reply:
This was implemented before the recent Core changes and is not needed anymore, as all the inputs have expected heights.